.github/workflows/static-analysis.yaml

```diff
@@ -0,0 +1,16 @@
+name: "Static code analysis"
+on:
+  pull_request_target:
+    branches: [ "master", "release/**" ]
+
+jobs:
+  static-analysis:
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
AFAIU with pull_request_target event this will check out the base repository, please see https://github.com/actions/checkout/issues/518 and https://dev.to/suzukishunsuke/secure-github-actions-by-pullrequesttarget-641

do you know how other projects are using it? I tried to search code on github, but came up empty

Reply:
updated following doc in https://github.com/tiobe/tics-github-action